javacord-api/src/main/java/org/javacord/api/entity/server/PremiumTier.java

```diff
@@ -0,0 +1,63 @@
+package org.javacord.api.entity.server;
+
+/**
+ * An enum with all premium tier levels (Server Boost level).
+ */
+public enum PremiumTier {
```

Comment:
I think we should name this `BoostLevel` instead.
Generally, Javacord uses the names used in the client and not the ones from the api (e.g., Server instead of Guild).
```suggestion
public enum BoostLevel {
```